auslib/web/public/api.yml

```diff
@@ -582,8 +320,94 @@ paths:
 
   /__version__:
     get:
-      operationId: "auslib.web.public.dockerflow.version"
+      operationId: 'auslib.web.public.dockerflow.version'
       description: Respond to /__version__ with the contents of /app/version.json.
       responses:
         200:
           description: version.json content.
+
+parameters:
+  productParam:
```

Comment:
Minor nit: There's no need to include "Param" at the end of each of these. Anytime they are referenced, they are proceeded by "paramaters" so it ends up being redundant.